Makefile.docker

```diff
@@ -0,0 +1,34 @@
+TOPDIR=$(dir $(lastword $(MAKEFILE_LIST)))
```

Comment:
```suggestion
# Makefile.docker contains the shared tasks for building, tagging and pushing Docker images.
# This file is included into the Makefile files which contain the Dockerfile files (E.g.
# kafka-base, kafka etc.).
#
# The DOCKER_ORG (default is name of the current user) and DOCKER_TAG (based on Git Tag,
# default latest) variables are used to name the Docker image. DOCKER_REGISTRY identifies
# the registry where the image will be pushed (default is Docker Hub).
TOPDIR=$(dir $(lastword $(MAKEFILE_LIST)))
```

I would leave the comments block we have in the Strimzi HTTP bridge to clarify what's defined here.